drivers/hmis/app/graphql/schema.graphql

```diff
@@ -404,12 +404,15 @@ type Client {
 
 type ClientAccess {
   canDeleteAssessments: Boolean!
+  canDeleteClient: Boolean!
   canDeleteEnrollments: Boolean!
+  canEditClient: Boolean!
   canEditEnrollments: Boolean!
   canManageAnyClientFiles: Boolean!
   canManageOwnClientFiles: Boolean!
   canViewAnyConfidentialClientFiles: Boolean!
   canViewAnyNonconfidentialClientFiles: Boolean!
+  canViewClient: Boolean!
```

Comment:
Are there cases where GraphQL would resolve a Client that has `canViewClient: false`? 🤔

Reply:
Probably not in our case. I'll remove it